SevenSlots/SevenSlots/SevenSlots/Commands/LoginCommand.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Android.Content;
+﻿//using Android.Content;
```

Comment:
why not just delete it?

Reply:
Meant to ask you about this import, but then I saw the program working without them as well and just forgot about them. I'll delete them.